apps/website/src/components/show-and-tell/ShowAndTellEntryForm.tsx

```diff
@@ -320,7 +289,11 @@ export function ShowAndTellEntryForm({
               label="Content"
               name="text"
               defaultValue={entry?.text}
-              maxLength={700}
+              maxLength={
+                entry?.createdAt && entry.createdAt < dateWhenTextShortened
+                  ? 700
+                  : 300
+              }
```

Comment:
Is this enforced at the DB at all (either length), or entirely client-side?

Reply:
So yes and no. We do a simple length check on the HTML (1k) on the server and check the text characters in client (300/700, ignoring html tags). 

I think this is the most simple way to do this. In theory this means that the text could be rejected server-side cause the tags (and attributes) even though its <= 300/700 chars in the client. Also people could "hack" the client and send up to 1k chars. But I think this is not actually a problem since we review the posts anyways.

I made a new PR to improve the formatting of server error messages #672

Reply:
Problem with checking text chars on the server is, we'd probably need to strip the tags and count chars, ideally with JSDOM, but we already use dompurify, meaning a lot of work for very little benefit.

Checking HTML length in the client on the other hand seems like bad UX...

Reply:
I moved the limit in the data file and added a comment. I hope that helps a bit :)